applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -332,9 +332,15 @@ static void syncable_cb(struct bt_audio_broadcast_sink *sink, bool encrypted)
 	strncpy(bis_encryption_key, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY, 16);
 #endif /* (CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED) */
 
-	if (active_stream.stream->ep->status.state == BT_AUDIO_EP_STATE_STREAMING ||
-	    !playing_state) {
-		LOG_DBG("Syncable received, but either in paused_state or already in a stream");
+	if (active_stream.stream->ep != NULL) {
+		if (active_stream.stream->ep->status.state == BT_AUDIO_EP_STATE_STREAMING) {
+			LOG_DBG("Syncable received, but already in a stream");
+			return;
+		}
+	}
```

Comment:
For consistency I suggest:
```suggestion
        if (active_stream.stream == NULL || active_stream.stream->ep == NULL) {
		LOG_DBG("No active stream");
		return;
	}
	
	if (active_stream.stream->ep->status.state == BT_AUDIO_EP_STATE_STREAMING) {
		LOG_DBG("Syncable received, but already in a stream");
		return;
	}
```

Reply:
Hi @alexsven, thank you for your feedback! The thing is that `active_stream.stream->ep` is `NULL` when this callback (`syncable_cb`) is called for the first time. `ep` gets allocated inside `bt_audio_broadcast_sink_sync` which is called a few lines below. So returning when `ep` is `NULL` would break the code. It is just necessary to make sure that `ep->status.state` is not checked if `ep` was not allocated yet. 

About checking `active_stream.stream` to be `NULL`, yes, let's add it. I didn't include it in the original PR, because when `syncable_cb` is called, it means that `base_recv_cb` must have been called before (at least in a happy flow scenario). `active_stream.stream` always gets assigned in `base_recv_cb`. Anyway, adding this check to `syncable_cb` cannot hurt and and it will make the code more robust in case of some weird error case.